entc/integration/config/ent/user.go

```diff
@@ -19,7 +19,10 @@ type User struct {
 	config `json:"-"`
 	// ID of the ent.
 	ID int `json:"id,omitempty"`
-	// Name holds the value of the "name" field.
+	// Age holds the value of the "age" field.
+	Age int `json:"age,omitempty"`
+	// Comment line1
+	//  Comment line2
 	Name string `json:"name,omitempty"`
```

Comment:
Make sure to test the generated comment in your test.

You can use the `ast`/`parser` packages as follows:

```go
f, err := parser.ParseFile(token.NewFileSet(), "ent/user.go", nil, parser.ParseComments)
ast.Inspect(f, func(n ast.Node) bool {
	if f, ok := n.(*ast.Field); ok && len(f.Names) > 0 && f.Names[0].Name == "Name" {
		require.Equal(t, descriptorComment, f.Doc.Text())
		return false
	}
	return true
})
``` 

It's a bit extreme. But this will make sure the behaviour won't change in the future.